README.md

```diff
@@ -1,70 +1,88 @@
-# Getting Started with Create React App
+# Project Information
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
-
-## Available Scripts
 
-In the project directory, you can run:
+    This project is frontend part of the STONKS application.
```

Comment:
Same to same nahi tape na tha. Paragraph mai bana deta. Documentation mai to english use kar le :)

Reply:
Are han maine bas copy kara tha ki yaha edit kardunga. usmein abhi points add kardunga aur bhi
same waise bhi na rakhta main